src/main/java/gov/usds/case_issues/model/CaseSnoozeSummaryFacade.java

```diff
@@ -44,6 +55,17 @@ public ZonedDateTime getSnoozeEnd() {
 		return wrapped.getSnoozeEnd();
 	}
 
+	public String getCreatedBy() {
+		return id;
+	}
+
+	public HashMap<String, String> getUser() {
```

Comment:
This should return a typed object rather than a map, for a silly-sounding but real reason: the API documentation doesn't know what the keys of this object are, but if we give it a POJO it'll know.

Reply:
:+1: this has the added benefit of ensuring we return the user object in the same format throughout the API